collab2/foraging/toolkit/subsampling.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+import numpy as np
+
+
+def subset_frames_evenly_spaced(df_raw, desired_frames=300):
+    df = df_raw.copy()
+    print("original_frames:", df["time"].max())
+    num_frames = df["time"].max()
+    print("original_shape:", df.shape)
+    df["time"] = np.round(df["time"] / (num_frames / desired_frames)).astype(int) + 1
+    df = df.drop_duplicates(subset=["time", "forager"], keep="first")
+    df = df[df["time"] <= desired_frames]
+    print("resulting_frames:", df["time"].max())
+    print("resulting_shape:", df.shape)
+
+    print("min_time", df["time"].min())
+    print("max_time", df["time"].max())
+
+    return df
+
+
+def rescale_to_grid(df, size):
```

Comment:
something's strange about `rescale_to_grid`. There should ultimately be only one version in `collab2` I think, and you have two of them. And I think you proposed passing max, but this isn't implemented in this one. Can you please try to resolve this?